src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java

```diff
@@ -66,6 +66,22 @@
  * <pre>
  * &lt;module name="NeedBraces"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * if (true) { // OK
```

Comment:
please reuse the same java code for ALL examples. To let user see how behavior is changing for the same code with and withtout config change.

Reply:
Done.